controls/organization.sp

```diff
@@ -49,12 +49,18 @@ control "org_two_factor_required" {
   tags        = local.organization_best_practices_common_tags
   sql = <<-EOT
     select
-      html_url as resource,
+      url as resource,
       case
+        when two_factor_requirement_enabled is null then 'info'
         when two_factor_requirement_enabled then 'ok'
         else 'alarm'
       end as status,
-      coalesce(name, login) || case when (two_factor_requirement_enabled)::bool then ' requires 2FA' else ' does not require 2FA' end || '.' as reason,
+      login || 
+        case 
+          when two_factor_requirement_enabled is null then ' insufficient permissions to check'
```

Comment:
I can't remember exactly, but with the v3 API, was it easy to tell when we had insufficient access on viewing 2FA required? If so, does the v4 API make it harder to distinguish this difference?

Reply:
This field is hydrated via v3 into the table, this merely expands the previous bool check to output a better `info` result when the user doesn't have the relevant permissions rather than an error.

Reply:
Maybe something like:
- `turbot require 2FA cannot be verified due to insufficient permissions.`
- `turbot requires 2FA.`
- `turbot does not require 2FA.`